{{cookiecutter.repo_name}}/package.json

```diff
@@ -444,7 +444,6 @@
       "at-rule-name-case": "lower",
       "at-rule-name-space-after": "always",
       "at-rule-no-vendor-prefix": true,
-      "stylelint-disable-reason": "always-before",
```

Comment:
I personally rather liked this rule. Is there a newer alternative to it, or a reason it is deprecated?